crates/rpc/src/cairo/ext_py/sub_process.rs

```diff
@@ -443,12 +463,15 @@ async fn process(
         (Command::EstimateFee { response, .. }, Ok(OutputValue::Fee(x))) => {
             let _ = response.send(Ok(x));
         }
+        (Command::SimulateTransaction { response, .. }, Ok(OutputValue::Traces(x))) => {
+            let _ = response.send(Ok(x));
+        }
         (command @ Command::Call { .. }, Err(fail))
-        | (command @ Command::EstimateFee { .. }, Err(fail)) => {
+        | (command @ Command::EstimateFee { .. }, Err(fail))
+        | (command @ Command::SimulateTransaction { .. }, Err(fail)) => {
```

Comment:
Could we just remove `@ Command::XXX { .. }` here too?

Reply:
Done